client/src/Components/Layouts/Customers/addcustomer.jsx

```diff
@@ -46,8 +46,8 @@ const CollectionCreateForm = Form.create({ name: 'form_in_modal' })(
           onCancel={onCancel}
           onOk={onCreate}
           className="modalcontainer"
-        >
-          <Form className='modalform'>
+        >modalform
```

Comment:
@Anies12 This looks like it might have been accidentally moved by your IDE, did you mean to move this "modalform" from being a classname to being plain text?

Reply:
Oh that's right :) , thank u